extension/packagemanagerextension/nbextension/common.js

```diff
@@ -0,0 +1,66 @@
+define([
+    'base/js/dialog',
+    'jquery'
+], function (dialog, $) {
+    "use strict";
+
+    /*
+    This function is responsible for the generation of confirmation modals.
+    */
+
+    function confirm(title, msg, button_text, callback) {
+        let buttons = { Cancel: {} };
+        buttons[button_text] = {
+            class: 'btn-danger btn-primary',
+            click: callback
+        }
+
+        let opts = {
+            title: title,
+            body: msg,
+            buttons: buttons
+        };
+
+        dialog.modal(opts);
+    }
+
+    /*
+    Auxiliary function for checking the DOM for selected packages.
+    */
+
+    function get_from_list(list) {
```

Comment:
I said you could use this, yes. I still think it's ok.
But you could also do it by adding an event to the "click" of the package. And the handler would add or remove the package from a list of selected values (kept in memory). Maybe this would be cleaner.